frontend/src/app/patients/personSchema.ts

```diff
@@ -259,10 +258,8 @@ const updateFieldSchemata: (
     ),
   gender: yup
     .mixed()
-    .oneOf(
-      [...getValues(GENDER_VALUES), "", null],
-      t("patient.form.errors.gender")
-    ),
+    .oneOf([...getValues(GENDER_VALUES), "", null])
```

Comment:
Sort of a nitpick, since the validation looks OK regardless, but fwiw we don't need to include an empty string or `null` as an allowable gender value since we're now requiring this field:

```suggestion
    .oneOf(getValues(GENDER_VALUES))
```